src/sentry/static/sentry/app/components/shortId.tsx

```diff
@@ -39,7 +45,9 @@ const StyledShortId = styled('div')`
   justify-content: flex-end;
 `;
 
-const StyledAutoSelectText = styled(AutoSelectText, {shouldForwardProp: isPropValid})`
+const StyledAutoSelectText = styled(AutoSelectText, {shouldForwardProp: isPropValid})<{
```

Comment:
Kinda a side comment here, but we can kill the `shouldForwardProp` here since `isPropValid` is  the default.

Reply:
thanks, removed it 👍